doc/whatsnew/fragments/7860.false_positive

```diff
@@ -0,0 +1,3 @@
+`multiple-statements` (`C0321`) now no longer triggers for function stubs using `...`.
```

Comment:
```suggestion
``multiple-statements`` (`C0321`) now no longer triggers for function stubs using ``...``.
```

Reply:
This is so that these phrases appear as code in the docs & not italics(as is the case with a single `)